vipps-report-api.md

```diff
@@ -7,14 +7,49 @@ sidebar_position: 3
 
 END_METADATA -->
 
+# Vipps Report API
```

Comment:
I don't agree with this; "Settlement != Report"

There may probably be a Report API guide in the future, talking more generally about how metadata flows from epayment into transaction reports etc. That is something else than the Settlement guide.